### ISSUE TITLE
Trim down data loaded into browsers

### ISSUE THREAD

**Issue by stefanb** [sledilnik/zdravniki#115]
Check which columns from https://github.com/sledilnik/zdravniki-data/tree/main/csv are not used and either use them (eg in search) or remove them from cvs

## [dict-institutions.csv](https://github.com/sledilnik/zdravniki-data/blob/main/csv/dict-institutions.csv):

* [ ] id_inst
* [ ] zzzsSt (will become a key, when we manage to match everything)
* [X] name
* [ ] unit ?
* [X] address
* [X] post
* [X] city
* [ ] municipality ?
* [X] lat,lon
* [X] phone
* [X] website

We could also add statistical region name if needed.

## [doctors.csv](https://github.com/sledilnik/zdravniki-data/blob/main/csv/doctors.csv):

* [ ] id (synthetic id, not really useful, best to not use it and remove it from CSV)
* [X] doctor
* [X] type
* [X] accepts
* [X] availability
* [X] load
* [X] id_inst

**Comment by lukarenko** [comment]
@stefanb yes, we should remove `id` if not used in web - just causes larger diff on updates

**Comment by stefanb** [comment]
@jalezi can you confirm that we aren't using `id` field from `doctors.csv`?

**Comment by jalezi** [comment]
> @jalezi can you confirm that we aren't using `id` field from `doctors.csv`?

It needs here we are using. But this is easy fix. Replacing with something unique as `doctor.key`.
![Screenshot 2021-12-20 12 00 24](https://user-images.githubusercontent.com/44704999/146757357-2edde0d2-9f4f-49d0-9e5a-fa96f5f08854.png)

I'll do it right now. There is a dedicated branch, right?

**Comment by jalezi** [comment]
@stefanb I also have to change 'doctorsContext.js`. This will take some time. I will explain a little bit later.
